torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2086,9 +2087,10 @@ Tensor slice_backward_wrapper(
 // This makes no assumption on the signs of sigma.
 Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const Tensor& self,
           bool some, bool compute_uv, const Tensor& raw_u, const Tensor& sigma, const Tensor& raw_v) {
+
   TORCH_CHECK(compute_uv,
-           "svd_backward: Setting compute_uv to false in torch.svd doesn't compute singular matrices, ",
-           "and hence we cannot compute backward. Please use torch.svd(compute_uv=True)");
+           "svd_backward: Setting compute_uv=False in torch.svd doesn't compute the singular vectors, ",
```

Comment:
Language nit:

"cannot compute the backward" -> "cannot compute the gradient"

also, shouldn't this now direct people to torch.linalg.svdvals?

Reply:
Sorry, this change does not belong in this PR. I reverted it.